spec/lib/scorecard_api/client_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'scorecard_api/client'
+
+describe ScorecardApi::Client do
+  let(:client) { described_class.new }
+
+  it 'gets a list of schools' do
+    allow(client).to receive(:perform).with(:get, 'schools', any_args).and_return(Faraday::Env.new)
+
+    params = {
+      'fields': ScorecardApi::Service::API_MAPPINGS.keys.join(','),
+      'per_page': ScorecardApi::Service::MAX_PAGE_SIZE.to_s,
+      'page': 0
+    }
+
+    client_response = client.schools(params)
+    expect(client_response).to be_an(Faraday::Env)
```

Comment:
All you are testing here is that `perform` gets called.  Everything else is stubbed.
This whole test could be:
```ruby
expect(client).to receive(:perform)
client.schools({})
```
This might be a good place for a VCR cassette, although given there's not a ton going on in this class.

Reply:
changed